packages/valaxy/node/utils/config.ts

```diff
@@ -71,3 +71,36 @@ export async function resolveValaxyConfig(options: ValaxyEntryOptions) {
     theme,
   }
 }
+
+export async function resolveValaxyConfigFromAddonRoot(root: string, addonOptions: ValaxyAddonResolver, valaxyOptions?: ResolvedValaxyOptions) {
```

Comment:
I think it can reuse `resolveValaxyConfigFromRoot` by pass file name. (default is `valaxy.config`)